src/components/Catalog.js

```diff
@@ -13,25 +13,16 @@ const providerItems = (
   selectedItemId,
   selectedProviderId,
   selectedTravelClassId,
-  setSelectedTravelClassId,
+  onSelectTravelClass,
```

Comment:
rename as onTravelClassSelect - guideline is on<Abc><Event>

Reply:
Fixed